src/test/TeleportRouter.t.sol

```diff
@@ -368,13 +365,13 @@ contract TeleportRouterTest is DSTest {
         router.settle("l2network", "another-l2network", 100 ether);
     }
 
-    function testSettleFromparentGateway() public {
-        router.file("parentGateway", address(this));
+    function testSettleFromParentGateway() public {
+        router.file("gateway", parentDomain, address(this));
         router.file("gateway", "another-l2network", address(new GatewayMock()));
         DaiMock(dai).mint(address(this), 100 ether);
         DaiMock(dai).approve(address(router), 100 ether);
 
-        router.settle("l2network", "another-l2network", 100 ether);
+        router.settle(parentDomain, "another-l2network", 100 ether);
```

Comment:
I think you probably meant `"l2network"` here instead of `parentDomain`, otherwise this test would be kind of redundant with `testSettleTargetingSubDomain()` above.

Reply:
Yes this should stay as `"l2network"` as it's undefined in the router, but since the request comes from the `parentDomain` it will be accepted with anything as the source.

Reply:
Fixed